### ISSUE TITLE
Markers not reachable using keyboard [Accessibility issue]

### ISSUE THREAD

**Issue by nidhimittal90** [d3/d3-geo#276]
We are using d3-geo to display worldmap with markers on few countries, while doing accessibility testing using [Accessibility Insights for Web](https://accessibilityinsights.io/docs/web/overview/) extension, none of the markers(countries) are reachable using tab key. This is a serious concern for accessibility as the interactive elements are not reachable through keyboard. Please let me know if there is any fix for this issue.
